spec/svg_spec.rb

```diff
@@ -10,7 +10,9 @@
       it 'does not contain inline style tags (that render poorly in IE due to CSP)' do
         doc = Nokogiri::XML(File.read(svg_path))
 
-        expect(doc.css('style')).to be_empty
+        expect(doc.css('style')).to be_empty.or(
+          have_attributes(text: match(/^\s*\/\*\s*test-ignore\s*\*\//)),
```

Comment:
WDYT of calling this `lint-ignore` instead of `test-ignore`?

Reply:
Also! What if we used the `%r||` syntax to reduce escaping those slashes:

```ruby
%r|^\s*/\*\s*test-ignore\s*\*/|
```

Reply:
> WDYT of calling this `lint-ignore` instead of `test-ignore`?

I'd actually gone back and forth on this very naming 😄  I don't feel strongly one way or the other. I'll update it.

> Also! What if we used the `%r||` syntax to reduce escaping those slashes:

TIL about this syntax. I think it reads much nicer 👍

Reply:
> WDYT of calling this `lint-ignore` instead of `test-ignore`?

Updated in 9bb29c2.

> Also! What if we used the `%r||` syntax to reduce escaping those slashes:
> 
> ```ruby
> %r|^\s*/\*\s*test-ignore\s*\*/|
> ```

Updated in ebfd2cd, with small caveat that our Rubocop configuration prefers `%r{}` over `%r||`:

```
spec/svg_spec.rb:14:39: C: Style/PercentLiteralDelimiters: %r-literals should be delimited by { and }.
          have_attributes(text: match(%r|^\s*/\*\s*lint-ignore\s*\*/|)),
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Ah right I had a feeling it would complain lol